docker-compose.yml

```diff
@@ -34,7 +34,17 @@ services:
       DB_BUILD: ${DB_BUILD}
       BLAST_HOST: ${BLAST_HOST:-blast}
       FORMATTER_HOST: ${FORMATTER_HOST:-formatter}
+
+      # Job Config
+      #
+      # These options configure the validation/management of jobs.
       JOB_TIMEOUT: ${JOB_TIMEOUT:-30} #days
+      MAX_QUERIES_PER_REQUEST: 100
```

Comment:
How does the client override `maxQueries`? It wasn't immediately clear to me from the code.

Reply:
Updated to include this.